fiftyone/core/utils.py

```diff
@@ -1562,7 +1567,7 @@ def get_default_batcher(iterable, progress=False, total=None):
             iterable,
             target_latency=target_latency,
             init_batch_size=1,
-            max_batch_beta=2.0,
+            max_batch_beta=8.0,
```

Comment:
Just want to make sure this is intended. 2.0 was what was used originally.

I changed it to 8.0 for the `BSONSizeDynamicBatcher` because I figured we're going to be more confident with the size based estimate than the latency based estimate (less variables), as well as the target batch size will likely need to be much larger than with the latency version. So we'd rather converge on the "proper" size faster and with less calls to MongoDB (going from batch size 1 to 10k takes 6 calls as opposed to 15)